Modules/ProfileManager/ProfileManager.lua

```diff
@@ -37,7 +37,7 @@ function ProfileManager:Initialize()
             local name = entry:name()
             if UTILS.empty(name) then return end
 
-            local class = UTILS.GetClassReadable(UTILS.NumberToClass(entry:ingameClass()))
+            local class = UTILS.NumberToClass(entry:ingameClass()) or ""
```

Comment:
I was also thinking:
can we actually store the class number also or only and use it whenever needed? aka: in the icon?